### PR TITLE
[KYUUBI #2453] [Improvement] checkValue of TypedConfigBuilder shall also print the config name

### PR DESCRIPTION
**Pull request by zhaomin1423**
### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by cxzl25** [comment]
remove title ` [Improvement]`

File: kyuubi-common/src/main/scala/org/apache/kyuubi/config/ConfigBuilder.scala [hunk 1]
```diff
@@ -136,7 +136,10 @@ private[kyuubi] case class TypedConfigBuilder[T](
   /** Checks if the user-provided value for the config matches the validator. */
   def checkValue(validator: T => Boolean, errMsg: String): TypedConfigBuilder[T] = {
     transform { v =>
-      if (!validator(v)) throw new IllegalArgumentException(errMsg)
+      if (!validator(v)) {
+        throw new IllegalArgumentException(s"The value of ${parent.key} is invalid," +
```
Comment (cxzl25) [inline kyuubi-common/src/main/scala/org/apache/kyuubi/config/ConfigBuilder.scala:140]: We can refer to [SPARK-35456](https://issues.apache.org/jira/browse/SPARK-35456), also output value.

nit space  `invalid,` ->`invalid, `

Comment (zhaomin1423) [reply kyuubi-common/src/main/scala/org/apache/kyuubi/config/ConfigBuilder.scala]: Thanks, updated.

**Review by cxzl25 — APPROVED** [review]

**Review by ulysses-you — APPROVED** [review]

**Comment by ulysses-you** [comment]
thanks, merging to master

**Pull request closed without merging**